src/languageHandler.js

```diff
@@ -64,75 +61,112 @@ export function _t(...args) {
 }
 
 /*
- * Translates stringified JSX into translated JSX. E.g
- *    _tJsx(
- *        "click <a href=''>here</a> now",
- *        /<a href=''>(.*?)<\/a>/,
- *        (sub) => { return <a href=''>{ sub }</a>; }
- *    );
+ * Translates text and optionally also replaces XML-ish elements in the text with e.g. React components
+ * @param {string} text The untranslated text, e.g "click <a>here</a> now to %(foo)s".
+ * @param {object} variables Variable substitutions, e.g { foo: 'bar' }
+ * @param {object} tags Tag substitutions e.g. { 'a': (sub) => <a>{sub}</a> }
```

Comment:
Might be worth spelling out when one would use vars vs tags here (ie. tags is necessary whenever you need i18ned text inside an HTML tag like `<a>` or `<i>`?) With `_tJsx` you also needed to use `_tJsx` if you wanted any custom markup in the i18ned text (eg. an `img`) but since your new `_t` uses the same code paths for both, presumably tags like `<img>` could equally be done with a regular `vars` substitution?